### ISSUE TITLE
Implement Google analytics tracking

### ISSUE THREAD

**Issue by erussey** [emory-libraries/aspace#14]

**Comment by rotated8** [comment]
I assume that ASpace has a field you add our GA tracking number to, and it does the rest.

**Comment by tmill29** [comment]
Collin to follow up with Emily

**Comment by rotated8** [comment]
Here's the code:
```
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-83TR9951KC"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-83TR9951KC');
</script>
```

**Comment by rotated8** [comment]
As [best I can find](https://archivesspace.atlassian.net/wiki/spaces/AC/pages/38502475/FAQ+Potential+Topics), it needs to go in `public/views/layout_head.html.erb` in our plugin.

**Comment by rotated8** [comment]
This tag is set up for the presumed PUI production URL and the dev PUI site. Once this work is merged and we can see it working, and before this ticket is closed, the dev site should be removed.

**Comment by rotated8** [comment]
This is ready for @backend or @frontend to pick up.

**Comment by rotated8** [comment]
This is ready for @emory-libraries/backend or @emory-libraries/frontend to pick up

**Comment by erussey** [comment]
I have checked in with @eporter23 and can confirm that Google Analytics is receiving data.

**Comment by eporter23** [comment]
We should leave this ticket open or create a new one to update the domain in GA to the final production URL.

**Comment by erussey** [comment]
Blocked until we deploy to the production site (ticket #101).  We are going to try using the same tracking code for both test and prod to make deployments easier.  Emily will test this ticket by seeing if we're getting data from both environments and creating a "view" for both test and prod.

**Comment by eporter23** [comment]
Discussed with @erussey that we will try to use the same tracking ID for both test and prod and then see if I can separate the two on the Google Analytics side.

**Comment by erussey** [comment]
The Monday deploy this week pushed configurations to both prod and test and we should be able to test google analytics on both environments this week.

**Comment by eporter23** [comment]
I confirmed the GA scripts are now in production and Google is receiving data. I switched the GA4 setting to monitor the production URL and will check again on Monday since configuration changes can take up to 24 hours to be applied.

**Comment by eporter23** [comment]
Also, a GA4 property only allows for one web data stream, which can only be assigned to one site url. So if the same code is used for both prod and test, we will need to filter out test environment usage, but this is doable.

**Comment by erussey** [comment]
Closing this ticket as the code is in place and working.  Will create new ticket to address reporting setup out of GA.
